docs/content/tools/hicPlotTADs.rst

```diff
@@ -32,7 +32,7 @@ section as `color`, `title`, etc.
    # the content of the section label (in the previous example 'hic') is irrelevant for
    # plotting and only used to tell the user when something goes wrong.
    # There are two exceptions for this, the [x-axis] and the [spacer] sections
-   # that use the secion label to determine the action.
+   # that use the section label to determine the action.
 
    [hic]
    file = hic.npz
```

Comment:
I think the file should not en in .npz

Also, since now the default format is .h5 can you change all occurences of .npz by .h5?